### DISCUSSION TITLE
GLPI Agent for Android

### DISCUSSION THREAD

**Discussion by tulhaum** [glpi-project/glpi-agent#529] [Q&A]
### Bug reporting acknowledgment

Yes, I read it

### Professional support

None

### Describe the bug

GLPI Agent on Android 13 doesn´t run in background, neither startup from system boot, nor execute scheduled task.

### To reproduce

1. Install GLPI Agent from Play Store on Android 13
2. Set up list servers
3. Mark to "Startup from system boot" in Advanced Settings
4. Enable "automatic inventory" and define the "frequency" in Schedule Task.
5. Restart the smartphone. GLPI Agent won´t startup automaticaly. If open manually, doesn´t run in background and every inventory only runs manually.

### Expected behavior

GLPI Agent for Android should run in background, send the inventory at the scheduled time and startup from boot system.

### Operating system

Other (See additional context below)

### GLPI Agent version

1.3

### GLPI version

10.0.9

### GLPIInventory plugin or FusionInventory for GLPI plugin version

GLPI Inventory v1.2.3

**Comment by g-bougard** [comment]
Can you answer this issue not related to perl glpi-agent @stonebuzz ?

**Comment by g-bougard** [comment]
Hi @tulhaum 

I moved your issue as a discussion as it is not related to this GLPI Agent project. The related project is really there: https://github.com/glpi-project/android-inventory-agent
But issues are not opened there, so I can't transfer your question.

I asked my colleague and he told me Android probably changed the service handling with this new release. The problem should be addressed in the future but we can't tell you when as the colleague in charge has not the time to work on this problem right now. Anyway [think to subscribe to GLPI](https://services.glpi-network.com/) and request our direct support to make this problem a priority.

**Reply by tulhaum** [reply]
Thank you so much for the attention.

**Comment by stonebuzz** [comment]
Hi @tulhaum 

To be honest, this is the first time this problem has come up.
It's possible that the Android 13 API has introduced a change in scheduling management.

(In the past) I had started a PR to rework this part.  
https://github.com/glpi-project/android-inventory-agent/pull/323

But as I didn't get any negative feedback on the feature (and due to lack of time) I closed it.

Unfortunately, I don't have any more time now, 
If it becomes urgent/critical for, you may consider taking out a subscription to speed up the fix.

Best regards
